appsflyer/src/main/java/com/tealium/remotecommands/appsflyer/AppsFlyerRemoteCommand.kt

```diff
@@ -194,32 +187,32 @@ open class AppsFlyerRemoteCommand : RemoteCommand {
         }
     }
 
-    fun setHost(payload: JSONObject) {
-        val host: String? = payload.optString(Host.HOST)
+    private fun setHost(payload: JSONObject) {
+        val host: String = payload.optString(Host.HOST)
         val hostPrefix: String = payload.optString(Host.HOST_PREFIX)
 
-        host?.let { hostName ->
+        if(host.isNotEmpty()) {
             if (hostPrefix.isNotEmpty()) {
-                tracker.setHost(hostName, hostPrefix)
+                tracker.setHost(hostPrefix, hostPrefix)
```

Comment:
this change doesn't look correct - should we be passing the `hostPrefix` in twice?

Reply:
Ah, good catch! Fixed.